src/main.rs

```diff
@@ -92,6 +93,27 @@ fn film_id_from_response(response: letterboxd::SearchResponse) -> Vec<String> {
         .collect::<Vec<String>>()
 }
 
+/// Get film ids response of list entries request.
+fn film_id_set_from_response(response: letterboxd::ListEntriesResponse) -> HashSet<String> {
+    response
+        .items
+        .into_iter()
+        .map(|entry| entry.film.id)
+        .collect::<HashSet<String>>()
+}
+
+fn create_update_request(film_ids: (Vec<String>, Vec<String>)) -> letterboxd::ListUpdateRequest {
```

Comment:
I tried currying to pass the list name but failed so far :/.

Reply:
This can be done below like that
```rust
.map(|film_ids| {
    create_update_request(String::from("to-watch"), film_ids)
})
```

Of course, the signature has to be then:
```rust
fn create_update_request(list_name: String, film_ids: (Vec<String>, Vec<String>)) -> ...
```

Reply:
I know but 
```rust
.map(create_update_request(String::from("to-watch")))
```

seems nicer :)